dev-requirements.txt

```diff
@@ -1,2 +1,3 @@
 codacy-coverage
-pytest
\ No newline at end of file
+pytest
```

Comment:
For `requirements.txt` or `dev-requirements.txt` you can add specific version pins so that you know anyone else installing this will have the same, working version that you were using. So like this in fence, where we have:
```
pytest-flask==0.10.0
pytest==3.2.3
```
(you can use these versions, or the likely more recent ones you've been working with)